docs/testrunner.md

```diff
@@ -0,0 +1,19 @@
+# Test Runner
+
+## RSpec 

```

Comment:
1. Establecer criterios.
2. Mostrar las opciones existentes
3. Elegir las herramientas más adecuadas según estos criterios.
Además, estamos hablando de dos herramientas: test runner y biblioteca de aserciones. Es importante que distingáis entre las dos. Una búsqueda ligera de test frameworks en Ruby da varios, que ni siquiera mencionas. También bibliotecas de aserciones, que tampoco mencionas. Es imprescindible que aprendáis primero lo necesario para tomar decisiones y luego que justifiquéis esas decisiones para pasar este objetivo.